src/main/webapp/app/exam/shared/events/exam-live-event.component.html

```diff
@@ -23,6 +23,33 @@
             </div>
             <jhi-working-time-change [oldWorkingTime]="workingTimeUpdateEvent.oldWorkingTime" [newWorkingTime]="workingTimeUpdateEvent.newWorkingTime" />
         </div>
+        <div *ngSwitchCase="ExamLiveEventType.EXAM_ATTENDANCE_CHECK">
+            <div jhiTranslate="artemisApp.exam.events.messages.examAttendanceCheck.description" class="wt-title"></div>
+            <table *ngIf="event.user" class="table table-borderless mx-auto">
+                <tbody>
+                    <tr>
+                        <th>{{ 'artemisApp.studentExamDetail.name' | artemisTranslate }}</th>
+                        <td>{{ event.user?.firstName }} {{ event.user?.lastName }}</td>
+                    </tr>
+                    <tr>
+                        <th>{{ 'artemisApp.studentExamDetail.login' | artemisTranslate }}</th>
+                        <td>{{ event.user?.login }}</td>
+                    </tr>
+                    <tr>
+                        <th>{{ 'artemisApp.studentExamDetail.email' | artemisTranslate }}</th>
+                        <td>{{ event.user?.email }}</td>
+                    </tr>
+                    <tr *ngIf="event.user?.visibleRegistrationNumber">
+                        <th>{{ 'artemisApp.studentExamDetail.matriculationNumber' | artemisTranslate }}</th>
+                        <td>{{ event.user?.visibleRegistrationNumber }}</td>
+                    </tr>
+                    <tr *ngIf="examAttendanceCheckEvent.text && examAttendanceCheckEvent.text.length > 0">
```

Comment:
![grafik](https://github.com/ls1intum/Artemis/assets/26540346/aa994d63-c75c-4fa5-83a6-a52e973a7300)

```suggestion
                    <tr *ngIf="examAttendanceCheckEvent.text">
```